extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -71,6 +71,13 @@ public function getTopContributorsData() {
 
 		$topContributorsDetails = $this->getContributorsDetails( $topContributors );
 
+		global $wgLang;
+		$redirect = 'redirect=' . urlencode( SpecialPage::getTitleFor( 'CommunityPage' )->getCanonicalURL() );
+		$uselang  = 'uselang=' . $wgLang->getCode();
```

Comment:
You should be able to avoid the global with `$this->getLanguage()` instead of `$wgLang`.

Also, might be slightly cleaner to build the query string once since it's the same for both links, i.e.
```php
$query = wfArrayToCGI( [
	'redirect' => $this->getTitle()->getCanonicalURL(),
	'uselang' => $this->getLanguage()->getCode(),
] );

$login = '<a href="https://www.wikia.com/signin?' . $query . '" // etc.
```

I'd also consider using the `Html` class (which handles escaping for you) to build a HTML string for the links rather than manual string concatenation, i.e.
```php
$login = Html::element(
	'a',
	[ 'href' => 'https://www.wikia.com/signin?' . $query ],
	$this->msg( 'communitypage-anon-login' )->plain()
);
```